### ISSUE TITLE
Wazuh modulesd error in multiple wauzh-manager servers

### ISSUE THREAD

**Issue by S733** [wazuh/wazuh#8326]
|Wazuh version v4.0.3 | Installed on ubuntu 18

Today I received the following alert from two wazuh mangers

```
Received From: wazuh-manager->/var/log/audit/audit.log
Rule: 80711 fired (level 10) -> "Auditd: process ended abnormally"
Portion of the log(s):

type=ANOM_ABEND msg=audit(1619090044.796:1255): auid=1001 uid=0 gid=121 ses=138 pid=7015 comm="wazuh-modulesd" exe="/var/ossec/bin/wazuh-modulesd" sig=11 res=1
audit.type: ANOM_ABEND
audit.id: 1255
audit.pid: 7015
audit.auid: 1001
audit.uid: 0
audit.gid: 121
audit.session: 138


audit.command: wazuh
audit.exe: /var/ossec/bin/wazuh
audit.res: 1

```

After that I lost connection to Kibna-app, when I restarted the manager services and then one manager api got connected, the other still showing the following error when selecting the api.

```
Error
3005 - Some Wazuh daemons are not ready yet in node "node01" (wazuh-modulesd->failed)
```

In wazuh log I coundn't find any errors. The following error was there in syslog of both the managers.

```
Apr 21 20:02:04 localhost kernel: [12568455.288477] wazuh-modulesd[25771]: segfault at 0 ip 000000000045f531 sp 00007f0fdd91aa40 error 4 in wazuh-modulesd[400000+e2000]
Apr 21 20:02:04 localhost kernel: [12568455.288485] Code: be 88 7f 4b 00 4c 89 e7 e8 9c df fe ff 85 c0 0f 85 c5 02 00 00 48 8b 7c 24 28 44 89 fa be 01 00 00 00 45 89 fd e8 ff 74 fa ff <48> 8b 55 00 48 8b 7c 24 28 45 31 c0 b9 ff ff ff ff be 02 00 00 00


Apr 21 21:56:13 localhost kernel: [397842.925949] wazuh-modulesd[2234]: segfault at 0 ip 000000000045f531 sp 00007fdf1ae27a40 error 4 in wazuh-modulesd[400000+e2000]
Apr 21 21:56:13 localhost kernel: [397842.925958] Code: be 88 7f 4b 00 4c 89 e7 e8 9c df fe ff 85 c0 0f 85 c5 02 00 00 48 8b 7c 24 28 44 89 fa be 01 00 00 00 45 89 fd e8 ff 74 fa ff <48> 8b 55 00 48 8b 7c 24 28 45 31 c0 b9 ff ff ff ff be 02 00 00 00
```

Any solution for this issue.

**Comment by Veshialle** [comment]
We are experiencing the same issue (wazuh-manager version 3.13.2).
We've tried to run in debug the modulesd component and we encountered **Segmentation Fault**.

As the OP We encountered the same problem through ALL our cypeer manager, no configuration changed so we're guessing the vulnerabilities downloaded are causing this issue.

Probably this doesnt help that much but it seems that disabling _vulnerability-detector_ prevent it from crashing modulesd.
We're not 100% sure that modulesd doesnt crash anymore because it's pretty "random" and we're monitoring it right now.

Of course it's not a solution but it permit us to manage agents via kibana plugin.
